src/analyses/tmpSpecial.ml

```diff
@@ -0,0 +1,155 @@
+(* Analysis that tracks which variables hold the results of calls to math library functions.
+  For each equivalence a set of lvals is tracked, that contains all lvals on which the arguments of the corresponding call depend, so an equivalence can be removed if one of the lvals is written.*)
+
+open GoblintCil
+open Analyses
+
+module Spec =
+struct
+  include Analyses.IdentitySpec
+
+  let name () = "tmpSpecial"
+  module ML = LibraryDesc.MathLifted
+  module LS = SetDomain.ToppedSet(Lval.CilLval) (struct let topname = "All" end)
```

Comment:
Could just use `Queries.LS` like everything else. That avoids the need for yet another lvalue conversion function (`resolve`). Queries give `Queries.LS`, so just using that avoids need for such conversions altogether.